### PR TITLE
Timeline: widget styles

### PR DESCRIPTION
**Pull request by TeyaVes**
needs: https://github.com/telerik/kendo-themes/pull/843
targets: https://github.com/telerik/kendo/issues/9874

### CODE REVIEW

File: packages/default/scss/timeline/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,388 @@
+.k-timeline {
```
Comment (PreslavKozovski) [inline packages/default/scss/timeline/_layout.scss]: Wrap this code in a module

File: packages/default/scss/timeline/_layout.scss [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+    width: 100%;
+    padding: 0 $timeline-padding-x;
+    margin: $timeline-padding-y 0;
```
Comment (PreslavKozovski) [inline packages/default/scss/timeline/_layout.scss]: Rename this variable.
...
A possible solution is to rename both variables to something like:
$timeline-spacing-y
$timeline-spacing-x

File: packages/bootstrap/scss/_variables.scss [hunk 3]
```diff
@@ -1392,6 +1392,65 @@ $spreadsheet-drawing-outline-width: 2px !default;
...
+$timeline-track-start-calc: calc(#{$timeline-track-arrow-width} - 2 * #{$timeline-track-border-width}) !default;
+$timeline-track-end-calc: calc(#{$timeline-track-arrow-width} - 2 * #{$timeline-track-border-width}) !default;
+$timeline-track-bg: $button-bg !default;
+$timeline-track-border-color: $timeline-track-bg !default;
+
+$timeline-track-event-min-width: 90px !default;
+$timeline-track-event-offset: 36px !default;
+
+$timeline-vertical-padding: 80px !default;
+$timeline-vertical-padding-with-dates-calc: calc(#{$timeline-track-event-min-width} + #{$timeline-padding-x}) !default;
```
Comment (PreslavKozovski) [inline packages/bootstrap/scss/_variables.scss]: I do not understand what this var name means. Could you rename it to a more meaningful one?

File: packages/default/scss/timeline/_theme.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
+.k-timeline {
```
Comment (PreslavKozovski) [inline packages/default/scss/timeline/_theme.scss]: Wrap this code in a module

File: packages/default/scss/timeline/_theme.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        background-color: $timeline-flag-background;
+        color: $timeline-flag-text;
+    }
+
+    .k-timeline-circle {
+        background-color: $timeline-circle-color;
+    }
+
+    .k-timeline-card {
+        .k-card-header {
```
Comment (PreslavKozovski) [inline packages/default/scss/timeline/_theme.scss]:
```suggestion
        .k-timeline-card .k-card-header {
```

File: packages/default/scss/timeline/_theme.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+            color: $card-text;
+        }
+    }
+
+    .k-timeline-date {
+        color: $text-color;
+    }
+
+    .k-timeline-arrow {
+        &.k-button:not(.k-flat):not(.k-bare):not(.k-outline):not(.k-overflow-button).k-state-disabled {
```
Comment (PreslavKozovski) [inline packages/default/scss/timeline/_theme.scss]:
```suggestion
        .k-timeline-arrow.k-button:not(.k-flat):not(.k-bare):not(.k-outline):not(.k-overflow-button).k-state-disabled {
```

File: packages/material/scss/timeline.scss [hunk 7]
```diff
@@ -0,0 +1,11 @@
+@import "variables";
+@import "mixins";
+
+
+// Dependencies
+@import "common/all";
+
```
Comment (PreslavKozovski) [inline packages/material/scss/timeline.scss]: Same

File: packages/bootstrap/scss/_variables.scss [hunk 8]
```diff
@@ -1392,6 +1392,65 @@ $spreadsheet-drawing-outline-width: 2px !default;
...
+$timeline-date-margin-bottom: 10px !default;
+
+$timeline-circle-width: 16px !default;
+$timeline-circle-height: 16px !default;
+$timeline-circle-color: $accent !default;
+
+$timeline-collapse-arrow-padding-x: $padding-x-sm !default;
+
+$timeline-event-width: 400px !default;
+$timeline-event-height: 600px !default;
```
[the remark this answers is not in the document]
Comment (TeyaVes) [reply packages/bootstrap/scss/_variables.scss]: These are the default widths and heights of the events. They can be changed from the configuration of the widget

**Pull request merged**